docs/config.md

```diff
@@ -27,6 +27,13 @@ Copy-Item ".\etc\ovn_k8s.conf" -Destination (New-Item "C:\etc" -Type container -
 
 ## Config values
 
+The config file contains common configuration options shared between the various
```

Comment:
@dcbw - I have started a man page with this material (PR243 docs/ovn_k8s.conf.5)